src/settings.rs

```diff
@@ -17,8 +20,17 @@ pub struct DnsSettings {
 
 impl DnsSettings {
   pub fn load(filename: String) -> Result<Self, Box<dyn Error>> {
-    let contents = fs::read_to_string(shellexpand::full(filename.as_str()).unwrap().to_string())
-      .expect("Aw man, there was an issue while opening the config file :(");
+
+    let filenames= [filename.as_str(), "./dns.config.yaml", "~/.config/simpledns/dns.config.yaml", "/etc/simpledns/dns.config.yaml"];
```

Comment:
I forgot to add this originally sorry :(

The filename param should be optional and if it is defined, we will only use that one. 

Helps avoid confusion cause if they mistype the filepath it shouldnt just work anyway with a different config.

Reply:
Done in 69ae99d85f302a15464ba83d2969445a02b7311f